docs/content/en/docs/references/cli/_index.md

```diff
@@ -73,6 +73,7 @@ End-to-end pipelines:
 
 Pipeline building blocks for CI/CD:
   build             Build the artifacts
+  test              Test the artifacts
```

Comment:
I realize you're just pattern matching against what we already have written here so you don't necessarily need to address in this PR, but I think we should try not to use the term `artifact` in our docs if we can avoid it. it's not really a concept that has meaning outside of skaffold, so it's fine to use in code (e.g. for function names) but we shouldn't use it when talking about real concepts in docs.

this could be something like "Run tests against images built by Skaffold" or something similar.

Reply:
Added a tracking issue - https://github.com/GoogleContainerTools/skaffold/issues/5148